src/services/source/nativeIntegration.ts

```diff
@@ -38,9 +38,12 @@ export class NativeIntegrationSourceService implements SourceService {
     const respList: SourceTransformationResponse[] = await Promise.all<FixMe>(
       sourceEvents.map(async (sourceEvent) => {
         try {
+          const newSourceEvent = sourceEvent as { headers: any };
+          const { headers } = newSourceEvent;
```

Comment:
Lets add a basic type for source events from server?

Reply:
Added